PyRDF/Node.py

```diff
@@ -36,24 +36,6 @@ def _call_handler(self, *args, **kwargs):
 
         return newNode
     
-    def _dfs(self, node, prev_object= None):
-        """
-        Do a depth-first traversal of the graph from the root
-
-        """
-        if not node.operation:
-            prev_object = node._tdf
-            self._graph_prune()
-
-        else:
-            ## Execution of the node
-            op = getattr(prev_object, node.operation.name)
-            node.value = op(*node.operation.args, **node.operation.kwargs)
-        
-        for n in node.next_nodes:
-            self._dfs(n, node.value)
-
-    
     def _graph_prune(self):
```

Comment:
`_graph_prune` should not be flagged as internal since it is called from another module (`CallableGenerator`). It should be then `graph_prune`

Reply:
:+1: